src/runtime/runtime.js

```diff
@@ -709,6 +721,29 @@ export class ConfiguredRuntime {
   /** @override */
   showOffers(opt_options) {
     return this.documentParsed_.then(() => {
+      if (opt_options['oldSku']) {
+        console.error(
+          'Use the showUpdateOffers method for subscription updates'
+        );
+        return new Error(
+          'Use the showUpdateOffers method for new subscription updates'
+        );
+      }
+      const flow = new OffersFlow(this, opt_options);
+      return flow.start();
+    });
+  }
+
+  /** @override */
+  showUpdateOffers(opt_options) {
+    if (!isExperimentOn(this.win_, ExperimentFlags.REPLACE_SUBSCRIPTION)) {
+      throw new Error('Not yet launched!');
+    }
+    return this.documentParsed_.then(() => {
+      if (!opt_options['oldSku']) {
+        console.error('Use the showOffers method for new subscribers');
```

Comment:
We don't use console.error() in PROD code. The `new Error("..")` is enough.

Reply:
Done!